src/protagonist/Engine/Ingest/IngestController.cs

```diff
@@ -51,6 +55,16 @@ public async Task<IActionResult> IngestAsset(CancellationToken cancellationToken
 
         return ConvertToStatusCode(message, result.Status);
     }
+    
+    /// <summary>
+    /// Retrieve allowed av options
+    /// </summary>
+    [HttpGet]
+    [Route("allowed-av")]
+    public IActionResult ReturnAllowedAvOptions()

```

Comment:
`ReturnAllowedAvOptions` makes sense as a name, but would `GetAllowedAvOptions` be slightly clearer and in-line with other methods seen throughout the DLCS?